### DISCUSSION TITLE
What does this mean? `"extends": "nx/presets/npm.json"`

### DISCUSSION THREAD

**Discussion by SimonCockx** [nrwl/nx#13679] [General]
I recently created a new workspace, and I noticed that the `nx.json` file now starts with `"extends": "nx/presets/npm.json"`. I can't find it documented anywhere. What does this do? Where can I find this `npm.json` file?

**Comment by alexn-s** [comment]
great question. i am looking for an answer too

**Comment by SimonCockx** [comment]
Following up on it here: https://github.com/nrwl/nx/issues/13680
